src/store/index.js

```diff
@@ -1,12 +1,25 @@
 import { createStore } from "vuex";
 import axios from "axios";
+const api = axios.create({
+  baseURL: process.env.VUE_APP_BASE_URL,
+});
+api.interceptors.request.use(config => {
```

Comment:
create API composable for Axios related tasks, this should not be inside the create store main file. @ShivanshSinghnnovative